### ISSUE TITLE
Provide json format for import of operations

### ISSUE THREAD

**Issue by ZeHiro** [bnjbvr/kresus#173]
In the README maybe, so that it is possible to import transactions prio to the use of kresus

**Comment by bnjbvr** [comment]
Indeed, thanks for the report. It's actually just a dumb JSON dump of the database, with simple indexes instead of the CouchDB indexes, so making this readme is just about telling what the fields in the models are. Would you be interested in writing this?
